src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -173,7 +174,9 @@ protected FileMonitoringController(FilePath ws) throws IOException, InterruptedE
 
         @Override public byte[] getOutput(FilePath workspace, Launcher launcher) throws IOException, InterruptedException {
             // TODO could perhaps be more efficient for large files to send a MasterToSlaveFileCallable<byte[]>
-            return IOUtils.toByteArray(getOutputFile(workspace).read());
+            try (InputStream is = getOutputFile(workspace).read()) {
```

Comment:
I think this fixed [JENKINS-43639](https://issues.jenkins-ci.org/browse/JENKINS-43639).